<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fakeap</class>
 <widget class="QWidget" name="fakeap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>119</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>附近可用接入点列表：</string>
   </property>
  </widget>
  <widget class="QTableView" name="fake_table">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>119</y>
     <width>470</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="fakelistButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>25</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>获取FakeAP列表</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="fake_num">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>附近可疑接入点总数：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
